bedrock/mozorg/templates/mozorg/about/history-details.html

```diff
@@ -17,7 +17,7 @@ <h1 class="title-shadow-box">{{ _('History of the Mozilla Project') }}</h1>
       {% trans coderush='http://clickmovement.org/coderush',
                sourcerelease='http://web.archive.org/web/20021001071727/wp.netscape.com/newsref/pr/newsrelease558.html'
       %}
-        The Mozilla project was <a href="{{ coderush }}">created in 1998</a>
+        The Mozilla project was created in 1998
```

Comment:
This page is localized, this means you'll need to wrap this change around l10n tags so that the old translation is kept while the new string is being localized. See https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/about/history.html#L103 as an example